deeposlandia/datasets/tanzania.py

```diff
@@ -195,15 +330,24 @@ class method genericity
         image_list = os.listdir(os.path.join(input_dir, "images"))
         image_list_longname = [os.path.join(input_dir, "images", l)
                                for l in image_list
-                               if not l.startswith('.')][:nb_images]
+                               if not l.startswith('.')]
+        nb_image_files = len(image_list_longname)
 
         logger.info("Getting %s images to preprocess..."
-                    % len(image_list_longname))
+                    % nb_image_files)
```

Comment:
you can remove the `%` and replace it by a comma

Reply:
Link with previous comment, I mark this discussion as resolved and keep the previous one opened.